pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -389,8 +389,19 @@ type OIDCSettings struct {
 	ExtraScopes   string `json:"extraScopes,omitempty"`
 }
 
+// +kubebuilder:validation:Enum=metadata,recommended,minimal
+
+type AuditPolicyPreset string
+
+const (
+	AuditPolicyMetadata    AuditPolicyPreset = "metadata"
+	AuditPolicyRecommended AuditPolicyPreset = "recommended"
+	AuditPolicyMinimal     AuditPolicyPreset = "minimal"
+)
+
 type AuditLoggingSettings struct {
-	Enabled bool `json:"enabled,omitempty"`
+	Enabled      bool              `json:"enabled,omitempty"`
+	PolicyPreset AuditPolicyPreset `json:"policyPreset,omitempty"`
```

Comment:
Because now `AuditPolicyPreset` from here and from crds package are two different types, you have to convert them manually in [clones.go](https://github.com/kubermatic/kubermatic/blob/61f27b5f1c46c90cc6d932b07c3919985389fa4b/pkg/install/crdmigration/clones.go#L409) otherwise it fails to compile like this:

```
 pkg/install/crdmigration/clones.go:409:70: cannot convert old.AuditLogging (type *"k8c.io/kubermatic/v2/pkg/crd/kubermatic/v1".AuditLoggingSettings) to type *"k8c.io/kubermatic/v2/pkg/apis/kubermatic/v1".AuditLoggingSettings 
```

Alternatively just don't sync APIs at all, we are doing this manually from time to time anyway.